packages/onchainkit/.storybook/preview.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import type { Preview } from '@storybook/react';
+import { OnchainKitProvider } from '../src/OnchainKitProvider';
+import { base } from 'wagmi/chains';
+import '../src/styles/index.css';
+
+function getApiKey() {
+  try {
+    return process.env.NEXT_PUBLIC_ONCHAINKIT_API_KEY;
+  } catch (err) {
+    console.error(err);
+    return undefined;
+  }
+}
+
+const preview: Preview = {
+  decorators: [
+    (Story) => (
+      <OnchainKitProvider
+        apiKey={getApiKey()}
```

Comment:
why this instead of 
```
process.env.NEXT_PUBLIC_ONCHAINKIT_API_KEY ?? undefined
```

?

Reply:
IIRC Storybook didn't have access to process.env. This way we can set that up and things will still work.

Honestly, we should've just let it be undefined, but since this is the MVP setup for Storybook, this seemed fine for now.